examples/recording_service/pluggable_storage/c++11/CMakeLists.txt

```diff
@@ -19,15 +19,17 @@ if(NOT BUILD_SHARED_LIBS)
     set(msg
```

Comment:
@lulivi for cases like these where a certain combination of the build isn't supported, it looks like we would just skip it. Is that what you intended?

Reply:
We want to skip them in order to build all the other examples. If this fails, no build would be done unless the correct build-link modes are selected for this example. It is intended.